src/routes/index.js

```diff
@@ -1,14 +1,16 @@
-const React = require('react');
+const { lazy: Lazy } = require('react');
 const Auth = require('./auth');
-const Layout = require('../containers/Layout');
+// const Layout = require('../containers/Layout');
+const Layout = require('../components/Layout');
```

Comment:
In case this was leftover from testing things out:
```suggestion
const Layout = require('../containers/Layout');
```

Reply:
This was just to make this branch render, but I properly addressed it in [the next PR](https://github.com/BigRoomStudios/strangeluv/pull/273/files#diff-268a45f8e3c5fba5953593271f7aaed3d91812853a17dea472417fcc17c84bfdL9-L11) when I actually hooked it up and made it work.

I figured we can waterfall that one into this one  to keep things tidy, but it's no big deal either way.